src/Events/Calendar_Embeds/Admin/Page.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * Calendar Embeds Admin Page.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Calendar_Embeds\Admin
+ */
+
+namespace TEC\Events\Calendar_Embeds\Admin;
+
+use TEC\Common\StellarWP\Assets\Asset;
+use TEC\Events\Calendar_Embeds\Calendar_Embeds;
+use Tribe__Events__Main as TEC;
+
+/**
+ * Class Page
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Calendar_Embeds\Admin
+ */
+class Page {
+	/**
+	 * The menu hook suffix.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $hook_suffix;
+
+	/**
+	 * Register the Calendar Embeds menu item.
+	 *
+	 * @since TBD
+	 *
+	 * @return int
+	 */
+	public function register_menu_item() {
+		$cpt               = get_post_type_object( TEC::POSTTYPE );
```

Comment:
Since we are registering a CPT in here, can we rename this to `$event_cpt` to avoid potential confusion?